webapp/channels/src/components/widgets/inputs/password_input/password_input.tsx

```diff
@@ -73,11 +78,7 @@ const PasswordInput = React.forwardRef((
             onChange={onChange}
             onBlur={onBlur}
             onFocus={onFocus}
-            placeholder={createMode ? (
-                formatMessage({id: 'widget.passwordInput.createPassword', defaultMessage: 'Choose a Password'})
-            ) : (
-                formatMessage({id: 'widget.passwordInput.password', defaultMessage: 'Password'})
-            )}
+            placeholder={placeHolder}
```

Comment:
These password and login input fix the issue of `Ensures buttons have discernible text `. See https://accessibilityinsights.io/info-examples/web/button-name/